src/Plugin/BundleSuggestion/AudioEmbed.php

```diff
@@ -1,74 +0,0 @@
-<?php
-
-namespace Drupal\stanford_media\Plugin\BundleSuggestion;
```

Comment:
Audio is going away?

Reply:
for now. I wanted to push that out until OEmbed is supported for audio. Soundcloud is an oembed provider, but the core system is hard coded at this time